internal/agent/interactive_install.go

```diff
@@ -149,6 +149,10 @@ func InteractiveInstall(debug, spawnShell bool, sourceImgURL string) error {
 	block, err := ghw.Block()
 	if err == nil {
 		for _, disk := range block.Disks {
+			// skip useless devices (/dev/ram, /dev/loop, /dev/sr, /dev/zram)
+			if strings.HasPrefix(disk.Name, "loop") || strings.HasPrefix(disk.Name, "ram") || strings.HasPrefix(disk.Name, "sr") || strings.HasPrefix(disk.Name, "zram") {
```

Comment:
Prefix, or suffix? ... Maybe I'm mistaken in assuming `disk.Name` has `/dev/` in it

Reply:
ah nice, thanks!